imports/plugins/core/catalog/client/components/publishControls.js

```diff
@@ -1,12 +1,10 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { Components } from "@reactioncommerce/reaction-components";
-import AppBar from "@material-ui/core/AppBar";
 import Button from "@material-ui/core/Button";
-import Toolbar from "@material-ui/core/Toolbar";
 import Typography from "@material-ui/core/Typography";
 import withStyles from "@material-ui/core/styles/withStyles";
 import { i18next } from "/client/api";
+import PrimaryAppBar from "../../../../../client/ui/components/PrimaryAppBar/PrimaryAppBar";
```

Comment:
Can we make this `"/imports/client/ui/components/PrimaryAppBar` instead of all the `..`'s?